platform/ios/app/MBXViewController.m

```diff
@@ -709,6 +724,79 @@ - (void)removeParkLayer
     [self.mapView.style removeLayer:parkLayer];
 }
 
+- (void)styleFilteredFill
+{
+    // set style and focus on Texas
+    [self.mapView setStyleURL:[NSURL URLWithString:@"mapbox://styles/justin/citc7ya7w00002jpa0z16cjah"]];
+    [self.mapView setCenterCoordinate:CLLocationCoordinate2DMake(31, -100) zoomLevel:3 animated:NO];
+
+    // after slight delay, fill in Texas
+    dispatch_after(dispatch_time(DISPATCH_TIME_NOW, (int64_t)(2 * NSEC_PER_SEC)), dispatch_get_main_queue(), ^
```

Comment:
Looked into this with a patch such as the following: 

https://gist.github.com/1ab15bce31631f5ba032a731bd0bed38

However, it happens too quickly, essentially immediately after the style finishing tiling in, causing continual flashing between that happening and then the filter style changes. I want to eliminate the possibility that anything is done as part of the initial style fill, hence the delay so that the user can clearly see things in one state, then the other, to confirm that the filtered styling is applied.